src/goblint.ml

```diff
@@ -36,6 +36,7 @@ let main () =
         else
           Analyses.empty_increment_data ()
       in
+      if get_bool "ana.autotune.enabled" && not (get_bool "incremental.load") then AutoTune.chooseConfig file;
```

Comment:
Instead of silently turning off autotune in incremental mode (which in an ideal world should also work), we could just have that combination of options produce an error whereever those other checks are.